Libs/MRML/Core/vtkMRMLViewNode.h

```diff
@@ -247,6 +291,39 @@ class VTK_MRML_EXPORT vtkMRMLViewNode
 
   /// Show the Frame per second as text on the lower right part of the view
   int FPSVisible;
+
+  /// Tracking GPU memory size (in MB), not saved into scene file
+  /// because different machines may have different GPU memory
+  /// values.
+  /// A value of 0 indicates to use the default value in the settings
```

Comment:
For now, I don't think we should associate this property with the view. 

What about adding an `availableGPUMemory` property to the `vtkMRMLApplicationLogic` or the `qSlicerCoreApplication`